app/lib/matador/components/job-data/JobDataPanel.tsx

```diff
@@ -0,0 +1,42 @@
+import { Divider, Title } from "@mantine/core";
+import { NavLink } from "react-router-dom";
+import { JobData } from "./JobData";
+
+interface JobDataPanelProps {
```

Comment:
export the type

Reply:
Fixed with 35c6421357701641e9eedeec5819dece439e234c